deploy/olm-catalog/konveyor-operator/v1.2.0/konveyor-operator.v1.2.0.clusterserviceversion.yaml

```diff
@@ -387,6 +387,13 @@ spec:
           - migclusters
           verbs:
           - '*'
+        - apiGroups:
```

Comment:
I think we should add this to the non-olm permissions in the ansible role in case someone does a non-olm OCP4 install.